### PR TITLE
Add workflow to create Pull Request for updated Gemfile.lock

### PR DESCRIPTION
**Pull request by Watson1978**
This PR will add workflow due to create Pull Request for updated Gemfile.lock

Whe you run this workflow with manually, it will create PR like https://github.com/Watson1978/fluent-package-builder/pull/28

### CODE REVIEW

File: .github/workflows/windows-bundle-update.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.2 # Use same Ruby version with fluent-package bundled
+      - uses: actions/checkout@v4
+      - name: Bundle Update
+        run: |
+          cd fluent-package
+          gem install bundler:2.3.27
+          bundle update -j 4
+      - uses: peter-evans/create-pull-request@v5
```
Comment (kenhys) [inline .github/workflows/windows-bundle-update.yml:20]: Are there alternative way to do without Market place action.
(e.g. gh command)

https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

secrets.GITHUB_TOKEN is allowed to create PR.

Comment (Watson1978) [reply .github/workflows/windows-bundle-update.yml]: I fixed that it does not use Market place action.

File: .github/workflows/maintenance-bundle-update.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+          cd fluent-package
+          gem install bundler:2.3.27
+          bundle _2.3.27_ update -j 4
+      - name: Create Pull Request
+        run: |
+          $today=Get-Date -Format "yyyy-MM-dd"
+          $branch="maintenance/bundle-update/${today}"
+
+          git config user.name github-actions
+          git config user.email github-actions@github.com
```
Comment (kenhys) [inline .github/workflows/maintenance-bundle-update.yml:32]: I've heard that github-actions@github.com is not recognized appropriate.

https://github.com/orgs/community/discussions/26560

Comment (kenhys) [inline .github/workflows/maintenance-bundle-update.yml:32]:
```suggestion
          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

File: .github/workflows/maintenance-bundle-update.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        run: |
+          cd fluent-package
+          gem install bundler:2.3.27
+          bundle _2.3.27_ update -j 4
+      - name: Create Pull Request
+        run: |
+          $today=Get-Date -Format "yyyy-MM-dd"
+          $branch="maintenance/bundle-update/${today}"
+
+          git config user.name github-actions
```
Comment (kenhys) [inline .github/workflows/maintenance-bundle-update.yml:31]:
```suggestion
          git config user.name "github-actions[bot]"
```

File: .github/workflows/maintenance-bundle-update.yml (hunk 2 above)
Comment (kenhys) [reply .github/workflows/maintenance-bundle-update.yml:32]: ref. 

https://github.com/actions/checkout?tab=readme-ov-file#push-a-commit-to-a-pr-using-the-built-in-token

README: Suggest user.email to be 41898282+github-actions[bot]@users.noreply.github.com
https://github.com/actions/checkout/pull/1707

Comment (Watson1978) [reply .github/workflows/maintenance-bundle-update.yml:32]: Thanks!

Comment (Watson1978) [reply .github/workflows/maintenance-bundle-update.yml]: https://github.com/fluent/fluent-package-builder/pull/748/commits/e23ae96aa41136e3599d6cb589e0faeb287e90fa

File: .github/workflows/maintenance-bundle-update.yml (hunk 3 above)
Comment (Watson1978) [reply .github/workflows/maintenance-bundle-update.yml]: https://github.com/fluent/fluent-package-builder/pull/748/commits/e23ae96aa41136e3599d6cb589e0faeb287e90fa

**Review by kenhys — APPROVED** [review]
LGTM.

**Comment by daipom** [comment]
Thanks!!

**Pull request merged**